metadatahandler.py

```diff
@@ -122,6 +122,9 @@ def __writeGamelistEntry__(self, gamelist, dosGame, game, genre, useGenreSubFold
         if conversionType == util.retropie:
             path = "./" + genre + "/" + util.getCleanGameID(dosGame,'.conf') if useGenreSubFolders \
                 else "./" + util.getCleanGameID(dosGame, '.conf')
+        elif (conversionType == util.batocera or conversionType == util.retrobat) and useLongFolderNames:
```

Comment:
here we can test for useLongFolderNames only as we will enforce that it is always false in the GUI before starting generation when we are not using batocera or retrobat conversion